### DISCUSSION TITLE
Monitoring WebSocket traffic form iOS

### DISCUSSION THREAD

**Discussion by j-h-a** [mitmproxy/mitmproxy#5245] [Q&A]
I've used mitmproxy for monitoring/modifying https traffic from iOS, but now I'm working on an iOS app using WebSockets and the usual setup doesn't seem to yield anything. I've set up the proxy, installed the profile and made some https requests to ensure it's working and I can see the requests, but when I run my app via the proxy, it fails to connect and I don't see anything appearing in mitmproxy.

I've tried searching around but many of the results I think are old and probably out of date, but I'm not sure what the status is or where to find an up-to-date guide. Can anyone confirm if this does actually work / is supported? And any suggestions of what might be going wrong or what I can try to get more information?

**Comment by gastonmorixe** [comment]
It doesn't seems to work. Neither does Charles nor Proxyman (not even using socks5).
